packages/design-system/src/components/Dialog/Dialog.tsx

```diff
@@ -76,12 +76,10 @@ export interface BaseDialogProps extends AnalyticsOverrideProps {
    */
   onEnter?(): void;
   /**
-   * This function needs to handles the state change of exiting (or deactivating) the modal.
-   * Maybe it's just a wrapper around `setState()`; or maybe you use some more involved
-   * Flux-inspired state management — whatever the case, this module leaves the state
-   * management up to you instead of making assumptions.
-   * That also makes it easier to create your own "close modal" buttons; because you
-   * have the function that closes the modal right there, written by you, at your disposal.
+   * Called when the user triggers an exit event, like by clicking the close
+   * button or pressing the ESC key. The parent of this component is
+   * responsible for showing or not showing the dialog, so this callback should
```

Comment:
might wanna remove "should" so we sound more certain of what this prop will do

Reply:
I'm prescribing what _they_ should use this prop to do. Maybe I need to make that more clear

Reply:
> so you need to use this callback to make that happen

How about that?